platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/isolated/IsolatedProjectsParallelConfigurationIntegrationTest.groovy

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.cc.impl.isolated
+
+import org.gradle.test.fixtures.server.http.BlockingHttpServer
+import org.junit.Rule
+
+class IsolatedProjectsParallelConfigurationIntegrationTest extends AbstractIsolatedProjectsIntegrationTest {
+
+    @Rule
+    BlockingHttpServer server = new BlockingHttpServer(5_000)
+
+    def setup() {
+        server.start()
+    }
+
+    def 'all projects are configured in parallel for #selector task selector'() {
```

Comment:
💅 The name of the test does not entirely fit anymore, now that we added a parameter to the task selector. We should either update the name or maybe split the `selector` column into two